kotlinpoet/src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -419,6 +443,7 @@ class TypeSpec private constructor(
     internal var primaryConstructor: FunSpec? = null
     internal var superclass: TypeName = ANY
     internal val initializerBlock = CodeBlock.builder()
+    var initializerIndex = -1
```

Comment:
Should be `internal`

Reply:
I kept it public because someone calling `toBuilder()` could want to change this order